comtypes/client/_generate.py

```diff
@@ -259,7 +262,7 @@ def _get_known_symbols():
     ):
         mod = importlib.import_module(mod_name)
         if hasattr(mod, "__known_symbols__"):
-            names = mod.__known_symbols__  # type: list[str]
+            names: List[str] = mod.__known_symbols__
```

Comment:
You should write type annotations by comments in the way of [PEP484](https://peps.python.org/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code).